.github/workflows/workflow_tests.yml

```diff
@@ -1,13 +1,34 @@
 name: tests
 
 on:
-  workflow_call
+  workflow_call:
+    inputs:
+      ci_type:
+        type: string
+        default: 'pr'
+      no_proxy:
+        type: string
+        default: 'localhost,127.0.0.1'
+      OPENAI_API_BASE:
+        type: string
+        default: 'http://localhost:8000/v1'
+      OPENAI_API_KEY:
+        type: string
+        default: '$your_openai_api_key'  
+      code_checkout_path:
+        type: string
+        default: '/home/runner/work/llm-on-ray/llm-on-ray'
 
 jobs:
   build:
```

Comment:
consider change the name build/build-docker to sth else such as bare-test/docker-test?